File: changsukuaidi-express-andriod/app/src/main/java/com/changsukuaidi/www/modules/login/launcher/LauncherActivity.java
package com.changsukuaidi.www.modules.login.launcher;

import com.changsukuaidi.www.base.BaseActivity;

/**
 * @Describe
 * @Author thc
 * @Date 2017/12/29
 */

public class LauncherActivity extends BaseActivity<LauncherFragment> {

    @Override
    protected LauncherFragment getFragment() {
        return new LauncherFragment();
    }

    @Override
    protected void initData() {

    }
}

File: WIFIDirect/app/src/main/java/com/thc/www/p2p/android/launcher/LauncherActivity.kt
package com.thc.www.p2p.android.launcher

import com.thc.www.p2p.base.BaseActivity

/**
 * @Describe
 * @Author thc
 * @Date  2018/5/7
 */
class LauncherActivity : BaseActivity<LauncherFragment>() {
    override fun getFragment(): LauncherFragment = LauncherFragment()
}